spanner/spanner_snippets/spanner/spanner_create_database_with_retention_period.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_create_database_with_version_retention_period]
+import (
+	"context"
+	"fmt"
+	"io"
+	"regexp"
+
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
+)
+
+func createDatabaseWithRetentionPeriod(w io.Writer, db string) error {
+	matches := regexp.MustCompile("^(.*)/databases/(.*)$").FindStringSubmatch(db)
+	if matches == nil || len(matches) != 3 {
+		return fmt.Errorf("Invalid database id %s", db)
+	}
+
+	ctx := context.Background()
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return err
+	}
+	defer adminClient.Close()
+
+	// Create a database with a version retention period of 7 days.
+	retentionPeriod := "7d"
+	alterDatabase := fmt.Sprintf(
+		"ALTER DATABASE `%s` SET OPTIONS (version_retention_period = '%s')",
+		matches[2], retentionPeriod,
+	)
+	op, err := adminClient.CreateDatabase(ctx, &adminpb.CreateDatabaseRequest{
```

Comment:
Mind declaring a separate `req` variable? Makes it a little easier to follow the call to `CreateDatabase`.

Reply:
Done.